ci-operator/step-registry/ovn/conf/custom-cidr/ovn-conf-custom-cidr-commands.sh

```diff
@@ -4,8 +4,8 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
-CLUSTER_NETWORK_CIDR=${CLUSTER_NETWORK_CIDR:-10.128.0.0/20}
-CLUSTER_NETWORK_HOST_PREFIX=${CLUSTER_NETWORK_HOST_PREFIX:-23}
+CLUSTER_NETWORK_CIDR=${CLUSTER_NETWORK_CIDR:-10.128.0.0/14}
+CLUSTER_NETWORK_HOST_PREFIX=${CLUSTER_NETWORK_HOST_PREFIX:-20}
```

Comment:
Please do not change the default here because that will impact other jobs using this step. Instead, please customize the CLUSTER_NETWORK_CIDR and CLUSTER_NETWORK_HOST_PREFIX as env in your own job.